tests/mech/test_mech_local.py

```diff
@@ -1,15 +1,18 @@
 import pytest
-from prediction_market_agent_tooling.benchmark.utils import OutcomePrediction
 
-from prediction_market_agent.tools.mech.utils import MechTool, mech_request_local
+from prediction_market_agent.tools.mech.utils import (
+    MechResponse,
+    MechTool,
+    mech_request_local,
+)
 from tests.utils import RUN_PAID_TESTS
 
 
 @pytest.mark.skipif(not RUN_PAID_TESTS, reason="This test costs money to run.")
 @pytest.mark.parametrize("mech_tool", list(MechTool))
 def test_mech_local(mech_tool: MechTool) -> None:
-    result: OutcomePrediction = mech_request_local(
-        question="Is the sky blue?",
+    response: MechResponse = mech_request_local(
+        question="Will the sun rise tomorrow?",
```

Comment:
`prediction-request-rag` mech not happy that `"Is the sky blue?"` is a yes/no question so throws exception 🤷